### PR TITLE
Removed ui/utils/prefs import from protocol/threads

### PR DESCRIPTION
**Pull request by bvaughn**
Follow on to #6979 and #6973.

### CODE REVIEW

File: packages/protocol/thread/thread.ts [hunk 1]
```diff
@@ -118,6 +117,12 @@ declare global {
   }
 }
 
+// Experimental feature
+let repaintAfterEvaluations: boolean = false;
+export function setRepaintAfterEvaluations(value: boolean): void {
+  repaintAfterEvaluations = value;
+}
```
Comment (bvaughn) [inline packages/protocol/thread/thread.ts:124]: This pattern is a tiny bit awkward I guess but at least it doesn't leak references to outside packages?

File: pages/_app.tsx [hunk 2]
```diff
@@ -134,6 +136,11 @@ if (isMock()) {
   configureMockEnvironmentForTesting();
 }
 
+// Expose app feature flags to the protocol through an app-agnostic API.
+if (features.repaintEvaluations) {
+  setRepaintAfterEvaluations(true);
+}
```
Comment (bvaughn) [inline pages/_app.tsx:142]: I assume this (during app initialization) is as good a place as any to set this logic?

Comment (markerikson) [reply pages/_app.tsx:142]: I'd probably lean towards putting it into `devtools.ts` with the rest of the recording-specific setup, but it'll work either way

**Review by markerikson — APPROVED** [review]

File: pages/_app.tsx (hunk 2 above)
Comment (bvaughn) [reply pages/_app.tsx:142]: Thanks! 👍🏼 

Looking at `devtools.ts` I don't see a place that stands out to me as better for it but I don't feel strongly at all if we move it there (or somewhere else)

**Pull request merged**